packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -655,33 +661,46 @@ describe('serializeAttribute ', () => {
     expect(serializeAttribute(node, NodePrivacyLevel.MASK, 'data-foo', DEFAULT_CONFIGURATION)).toBe('***')
   })
 
-  it('does not mask the default programmatic action name attributes', () => {
-    const node = document.createElement('div')
-    node.setAttribute(DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, 'foo')
-    expect(
-      serializeAttribute(node, NodePrivacyLevel.MASK, DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, DEFAULT_CONFIGURATION)
-    ).toBe('foo')
-  })
+  describe('attributes used to generate CSS selectors', () => {
+    it('does not mask the default programmatic action name attributes', () => {
+      const node = document.createElement('div')
+      node.setAttribute(DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, 'foo')
+      expect(
+        serializeAttribute(
+          node,
+          NodePrivacyLevel.MASK,
+          DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE,
+          DEFAULT_CONFIGURATION
+        )
+      ).toBe('foo')
+    })
 
-  it('does not mask the user-supplied programmatic action name attributes when it is a data attribute', () => {
-    const node = document.createElement('div')
-    node.setAttribute('data-testid', 'foo')
-    expect(
-      serializeAttribute(node, NodePrivacyLevel.MASK, 'data-testid', {
-        ...DEFAULT_CONFIGURATION,
-        actionNameAttribute: 'data-testid',
-      })
-    ).toBe('foo')
-  })
+    it('does not mask the user-supplied programmatic action name attributes when it is a data attribute', () => {
+      const node = document.createElement('div')
+      node.setAttribute('data-testid', 'foo')
+      expect(
+        serializeAttribute(node, NodePrivacyLevel.MASK, 'data-testid', {
+          ...DEFAULT_CONFIGURATION,
+          actionNameAttribute: 'data-testid',
+        })
+      ).toBe('foo')
+    })
 
-  it('does not mask the user-supplied programmatic action name attributes when it not a data attribute', () => {
-    const node = document.createElement('div')
-    node.setAttribute('testid', 'foo')
-    expect(
-      serializeAttribute(node, NodePrivacyLevel.MASK, 'testid', {
-        ...DEFAULT_CONFIGURATION,
-        actionNameAttribute: 'testid',
-      })
-    ).toBe('foo')
+    it('does not mask the user-supplied programmatic action name attributes when it not a data attribute', () => {
+      const node = document.createElement('div')
+      node.setAttribute('testid', 'foo')
+      expect(
+        serializeAttribute(node, NodePrivacyLevel.MASK, 'testid', {
+          ...DEFAULT_CONFIGURATION,
+          actionNameAttribute: 'testid',
+        })
+      ).toBe('foo')
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we should probably not use testid anymore in these tests since it now overlaps with stable attributes